README.md

```diff
@@ -1,6 +1,7 @@
 This README is under construction as we work to build a new community driven high performance key-value store.
 
 This project was forked from the open source Redis project right before the transition to their new source available licenses.
+Redis is an in-memory database that persists on disk. The data model is key-value, but many different kind of values are supported: Strings, Lists, Sets, Sorted Sets, Hashes, Streams, HyperLogLogs, Bitmaps.
```

Comment:
I think the better point here is that the reference that is a fork is way to high in our README, maybe we should just move it lower and explain what valkey is first. I would be open to updating that section.

Reply:
maybe we should just leave a hyperlink to redis.io? I think redis.io has done a good explanation job.

Reply:
Do we really want to link to redis.io though?

Reply:
I don't see that being problem. It is just a Google search away anyways. That said, neither explaining what redis is nor adding redis.io link is needed IMO. But if the concern is "lack of context", I would vote for simplicity. It is not our job to describe what Redis is (or isn't).

Reply:
If I didn't already know all about Redis, I would not appreciate being sent to read a link to Redis for a description of Valkey.  It would make be doubt that Valkey was ready for serious work.  Especially if that link were still being used two years from now.

It might be helpful to have a HISTORY section near the bottom.

Reply:
My suggestion was not taken how it was intended.  I'm not suggesting giving the history or the back story to Redis TMDR.  But I didn't know what the heck "high performance key-value store" is. When I use "key value" I'm literally adding black ink to the mix.  But I understood the Redis description well enough to know what type of software it is. 
Since this fork is taken from Redis, I took the Redis description of what Redis does as a quick copy and paste.  Just delete the "Redis" on line 4 and problem solved.